hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/request/snapshot/TestOMSnapshotCreateRequest.java

```diff
@@ -326,11 +334,42 @@ private void renameKey(String fromKey, String toKey) throws IOException {
         new OMKeyRenameResponse(omResponse, fromKeyInfo.getKeyName(),
             toKeyInfo.getKeyName(), toKeyInfo);
 
-    Assert.assertTrue(omMetadataManager.getSnapshotRenamedKeyTable().isEmpty());
     omKeyRenameResponse.addToDBBatch(omMetadataManager, batchOperation);
     omMetadataManager.getStore().commitBatchOperation(batchOperation);
   }
 
+  private void renameDir(String fromKey, String toKey, long offset)
+      throws Exception {
+    String fromKeyParentName = UUID.randomUUID().toString();
+    OmKeyInfo fromKeyParent = OMRequestTestUtils.createOmKeyInfo(volumeName,
+        bucketName, fromKeyParentName, HddsProtos.ReplicationType.RATIS,
+        HddsProtos.ReplicationFactor.THREE, 100L);
+
+    OmKeyInfo toKeyInfo = addKey(fromKey, offset + 4L);
```

Comment:
This need to be 
OmKeyInfo toKeyInfo = addKey(**toKey**, offset + 4L); 
OmKeyInfo fromKeyInfo = addKey(**fromKey**, offset + 5L); ?

Reply:
Thanks for pointing it out. Updated it.